pcmanfm/tabpage.h

```diff
@@ -217,6 +217,9 @@ protected Q_SLOTS:
     void freeFolder();
     QString formatStatusText();
 
+    // Adds bidi marks (RLM/LRM/RLE/LRE/POP) around the text for the statusbar.
+    QString encloseWithBidiMarks(QString& text);
+
```

Comment:
A `const` is missing from the argument, which results in:
```
error: binding reference of type ‘QString&’ to ‘const QString’ discards qualifiers
```
Please always try to compile the branch before committing! Thanks!

Reply:
I did, but forgot about this pointers thing!
_I actually can't understand pointers to the deep :)_

Reply:
It's good. I'll fix small problems after merging but I can't find any now.

Reply:
Oh, found one, which isn't small. This:
```cpp
(!layoutDirection() == Qt::LeftToRight) ? "\u200f" : "\u200e"
```
You mean the following; don't you?
```cpp
layoutDirection() == Qt::RightToLeft ? "\u200f" : "\u200e"
```
Please fix them and test the result!